functions.php

```diff
@@ -35,18 +35,30 @@ function( $template ) {
 
 require_once( __DIR__ . '/classes/class-p4-master-site.php' );
 
+$services = [
+	'P4_Metabox_Register',
+	'P4_Custom_Taxonomy',
+	'P4_Campaigns',
+	'P4_Post_Campaign',
+	'P4_Settings',
+	'P4_Control_Panel',
+	'P4_Post_Report_Controller',
+	'P4_Cookies',
+	'P4_Dev_Report',
+];
+
+// Load `P4_Campaign_Exporter` class on admin campaign listing page and campaign export only.
+if ( is_admin() && ( 'campaign' === filter_input( INPUT_GET, 'post_type', FILTER_SANITIZE_STRING ) || 'export_data' === filter_input( INPUT_GET, 'action', FILTER_SANITIZE_STRING ) ) ) {
+	array_splice( $services, 4, 0, 'P4_Campaign_Exporter' );
```

Comment:
Do we need to add them in specific order? Can we just `$services[] = 'P4_Campaign_Exporter'` for simplicity?
Also, please add both in 1 `if ( is_admin() )` since we will be adding more classes under this `if` check.

Reply:
I am not sure about the order of classes will impact or not, I will test and confirm it.